controllers/machine/machine_controller_test.go

```diff
@@ -3,27 +3,27 @@ package machine
 import (
 	"time"
 
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
```

Comment:
any reason we're returning to the default name of the package here? Makes it less readable

Reply:
no, I was just fixing the lint errors complaining :D 
I did not want to touch it more than necessary or disable the lint check